queries.sql

```diff
@@ -8,3 +8,42 @@ SELECT name, escape_attempts from animals WHERE weight_kg >10.5;
 SELECT name from animals WHERE neutered IS TRUE;
 SELECT name from animals WHERE name NOT IN ('Gabumon');
 SELECT name from animals WHERE weight_kg BETWEEN 10.4 and 17.3;
+BEGIN;
+	UPDATE animals SET species='unspecified';
+	ROLLBACK;
```

Comment:
I have seen from other TRANSACTION BLOCKS that you understand the task at hand. I will therefore not ask for required changes but just remind you that missed working on these 👇🏼:
- Before ROLLBACK an SQL statement to **_`verify that change was made`_** is expected, please add it.
- After ROLLBACK an SQL statement to **_`verify that change was undone`_** is expected, please add it.